src/eoapi/vector/eoapi/vector/sql/search.sql

```diff
@@ -0,0 +1,116 @@
+CREATE OR REPLACE FUNCTION search_items(
```

Comment:
modified version of https://github.com/stac-utils/pgstac/blob/9fc50271d73d35ba077184ccc797b1d8e730047b/sql/006_tilesearch.sql#L3-L118